### DISCUSSION TITLE
Artwork not showing up while locked or in notifications window (iPhone)

### DISCUSSION THREAD

**Discussion by ciu11o** [TortugaPower/BookPlayer#947] [Q&A]
I was wondering if there is something I need to do to enable to artwork I uploaded for the book to show up when my phone is locked. As of now it shows the player rectangle with all the normal functionality but the icon is just the default blue square.

![IMG_6150](https://github.com/TortugaPower/BookPlayer/assets/130109196/e81d55d0-0a84-4209-9937-8cb9dae84d13)

**Comment by GianniCarlo** [comment]
Hi @ciu11o ! I have some follow up questions to understand better the scenario:

- Was the artwork uploaded for the folder? If so, was it converted into a Volume in-app? (Just a heads up that progress between volume and individual mp3s are stored in different places). 
- If the artwork was uploaded for the individual mp3, is that showing properly inside the app in the player screen?
- Could you share a screenshot of the folder in-app? just to understand the file hierarchy

**Comment by ciu11o** [comment]
Thanks for such a fast response!

I think I did figure out what’s going on. But perhaps you can provide an
easy work around.

Dune, for example, I have it as 78 separate MP3’s (irritatingly haha). I
imported them into BookPlayer in a folder called Dune and it was easy to
start playing it. I then combined them all into a volume.

I uploaded the same artwork for both the folder and the volume of combined
MP3 files, but it still just showed up as a default blue square. I
realized, once I imported a MP3 file of another book (which was 1 single
file) the artwork worked fine.

So I separated the volume into the 78 individual MP3 files, and when I
uploaded artwork for each file, then it started showing up but only for the
files/chapters I uploaded artwork for (I just did this for the first 3 cuz
it was tedious).

Attached are some screenshots.

*So my understand is that the only way for artwork to show properly on a
locked player screen it for it to be uploaded for the file itself, not the
folder it’s in or the volume they’re combined into, correct?*

* Is there any way to quickly upload artwork for multiple files in a
folder/volume? I have a lot of books that are unfortunately in this format,
several MP3 files comprising the full book.*

Thank you! Your app is awesome and simple.




On Fri, Jun 30, 2023 at 2:36 PM Gianni Carlo ***@***.***>
wrote:

> Hi @ciu11o <https://github.com/ciu11o> ! I have some follow up questions
> to understand better the scenario:
>
>    - Was the artwork uploaded for the folder? If so, was it converted
>    into a Volume in-app? (Just a heads up that progress between volume and
>    individual mp3s are stored in different places).
>    - If the artwork was uploaded for the individual mp3, is that showing
>    properly inside the app in the player screen?
>    - Could you share a screenshot of the folder in-app? just to
>    understand the file hierarchy
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/TortugaPower/BookPlayer/discussions/947#discussioncomment-6328910>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/A7AU6DDAVAB5FUXDJWFFBRDXN4TFHANCNFSM6AAAAAAZ2EB5YQ>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>
-- 
*- Gregory Ciullo <https://www.linkedin.com/in/gregoryeciullo>*
